packaging/linux/run_keybase

```diff
@@ -45,7 +45,7 @@ echo Launching keybase service...
 keybase service &>> "$logdir/keybase.service.log" &
 echo Mounting /keybase...
 kbfsfuse -debug -mdserver mdserver.kbfs.keybase.io:443 \
-  -bserver bserver.kbfs.keybase.io:443 /keybase &>> "$logdir/keybase.kbfs.log" &
+  -bserver bserver.kbfs.keybase.io:443 -log-to-file /keybase &>> "/dev/null" &
```

Comment:
@cjb yes, what I meant to dream-about-but-not-quite-suggest was that if we sent unexpected output to some other file, we might want to make sure `keybase log send` includes that other file if/when it runs.